src/main/scala/no/gdl/bookapi/controller/OPDSController.scala

```diff
@@ -128,9 +126,9 @@ trait OPDSController {
                 <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=${currentPage - 1}"} rel="previous"/>
                 <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=${currentPage + 1}"} rel="next"/>
                 <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=$lastPage"} rel="last"/>
-        case OnlyOnePage(Paging(currentPage, currentPageSize)) =>
-                <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=$currentPage"} rel="first"/>
-                <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=$currentPage"} rel="last"/>
+        case OnlyOnePage(Paging(_, currentPageSize)) =>
+                <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=1"} rel="first"/>
+                <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=1"} rel="last"/>
```

Comment:
It's ok to keep whatever page-size the user entered, but these links shouldn't contain invalid page numbers.